Fix feature documentations (#325)
inveniosoftware/docs-invenio-rdm: docs/features/scalable.md

## Any file format/size

> InvenioRDM accepts any file format in any size given that your underlying storage infrastructure can support it.

Reviewer: ```suggestion
InvenioRDM accepts files of any format and any size that your underlying storage infrastructure supports.
```

## Versioning support

> Records and files are all versioned and our versioning feature performs automatic deduplication: so when 10KB of files change in a 100TB dataset we only store the new 10KB.

Reviewer: ```suggestion
Records and files are all versioned and our versioning feature performs automatic de-duplication: when a 10KB file among a 100TB dataset changes, only the new 10KB file is stored.
```
